### ISSUE TITLE
Don't recurse into .git

### ISSUE THREAD

**Issue by siraben** [astro/deadnix#1]
I ran this on my checkout of Nixpkgs and got the line

```
Error parsing file ./.git/logs/refs/remotes/upstream/revert-116335-move-aliases.nix: unexpected end of file
```

**Comment by astro** [comment]
May I generalize the option into "do(n't) descend into hidden dirs/files"?

**Comment by siraben** [comment]
Sounds good to me

**Comment by astro** [comment]
Fixed in 1a06f0c
